### ISSUE TITLE
Implement Army Combat Fitness Test Tracking & Scoring

### ISSUE THREAD

**Issue by mattgwagner** [mattgwagner/Battery-Commander#403]

**Comment by mattgwagner** [comment]
@erichmccartney I finished scoring on this tonight, but it does not take into account the Soldier's level/category for applying pass/fail yet.

**Comment by erichmccartney** [comment]
hooah, you're an animal.  everyone is Gold standard for next 2 years no?  oh wow this maybe the forcing function to finally get MOS in here.

**Comment by mattgwagner** [comment]
@erichmccartney yeah, the gold standard pass for the near-term is probably going to let me roll out the rest. 

As far as MOS, I was actually thinking of a different approach because it would have to look at MOS+rank. Have each Soldier include a property for their required level that is the template and is copied into the test instance, e.g. in case someone's required level changes after completing a test because they were promoted to O-3, the old test should still reflect pass/fail based on those standards.

**Comment by erichmccartney** [comment]
yeah, lots of changes here. the new scorecard seems to have mos on it. but I can see out this could cause issues for you in programming with promotion, your idea seems simpler
![image](https://user-images.githubusercontent.com/29133992/88822070-6e58d500-d191-11ea-8cc5-b1bee4b791a4.png)
